core/src/blob.rs

```diff
@@ -147,11 +138,7 @@ macro_rules! align {
 }
 
 pub const BLOB_HEADER_SIZE: usize = align!(SIZE_RANGE.end, BLOB_DATA_ALIGN); // make sure data() is safe for erasure
-pub const SIGNABLE_START: usize = PARENT_RANGE.start;
-
-pub const BLOB_FLAG_IS_LAST_IN_SLOT: u32 = 0x2;
-
-pub const BLOB_FLAG_IS_CODING: u32 = 0x1;
+pub const SIGNABLE_START: usize = SIGNATURE_RANGE.end;
```

Comment:
We probably don't need them to support signing either. For gossip, the CrdsValue contained within is signed.